scheduler/src/cook/mesos/util.clj

```diff
@@ -167,22 +167,34 @@
 
 (timers/deftimer [cook-mesos scheduler get-pending-jobs-duration])
 
+(defn get-pending-job-ents*
```

Comment:
Yea, perhaps it could just use a better name (in additional to making it private). Any suggestions on name?